tests/suites/0_stateless/03_0002_select_aggregator_with_filter.sql

```diff
@@ -1,2 +1,9 @@
 SELECT max(number) FROM numbers_mt (10) where number > 99999999998;
 SELECT max(number) FROM numbers_mt (10) where number > 2;
+
+select min(number) from numbers_mt (10) where 1 = 2;
```

Comment:
What happens to the following queries?
- `SELECT min(number) FROM numbers_mt(10) WHERE 1 = 2 GROUP BY number % 3`
- `SELECT number % 3, min(number) FROM numbers_mt(10) WHERE 1 = 2 GROUP BY number % 3`

Reply:
MySQL 5.7 behavior:
https://www.db-fiddle.com/f/qDnLWye3pM1thKfx4d8Z2M/1

Reply:
No row returns. I will add related test later.